### ISSUE TITLE
How to put service fabric mesh application into virtual network

### ISSUE THREAD

**Issue by pr3dr46** [Azure/service-fabric-mesh-preview#362]
I have a simple Service Fabric Mesh application with two microservices. When I deploy it I get public IP and I can access services.

What I would like is to wrap that Mesh application with Azure API management and restrict access to Mesh application. In other words, I would like to be able to use microservices only through API and disable direct access to them.

One idea is to put Mesh application and API into Virtual Network and add Application Gateway that will be visible public.

But I do not know how to put Mesh application into VNet.

**Comment by mattrowmsft** [comment]
We don't support this scenario quite yet (internally we call it bring your own vnet byovnet).  It is coming soon though.

**Comment by wiltwitt** [comment]
When can we expect this feature? We are trying to publish our apps to production.

**Comment by mattrowmsft** [comment]
You shouldn't put production workload on the service unless you have explicitly received a prod license from someone on containers team at Microsoft.

BYOVnet would appear in < 2 months.

**Comment by heymega** [comment]
@mattrowmsft - When it is released in 2 months time, will it have the same limitations as the BYOVnet has with Azure Container Instances?

I'd imagine it probably would given the service delegations appears to be the same (delegating a subnet to a ACI/SF Mesh).

We were told that BYOVnet would be available October 2018 and that SF Mesh would be GA January 2019 :'(

Whilst I still remain positive about its roll out in <2 months I can't help but feel apprehensive and feel like it will be rolled out in an immature state.

**Comment by mattrowmsft** [comment]
Yes, the experience will be same as what is available today with ACI but would be available for Windows (ACI only supports Linux today).

I can tell you that commits towards enabling the feature are already live for us.  I think post //BUILD (almost there) the timing of things will make more sense.

**Comment by brechtvhb** [comment]
Are there any updates on the release timeframe?
